src/routes/auth.routes.ts

```diff
@@ -10,22 +10,27 @@ import {
 } from "../controllers";
 import { ValidateJoi } from "../middleware/validation/validation.middleware";
 import { UserSchema } from "../middleware/validation/user.schema.middleware";
-import { isAuth } from "../middleware/auth";
+import { protectRoute } from "../services/protectRoutes.service";
+import { isSeller } from "../middleware/auth";
 
 const authRouter = Router();
 
+authRouter.post("/login", ValidateJoi(UserSchema.loginData.create), loginUser); // login a User
```

Comment:
```suggestion
authRouter.post("/login", ValidateJoi(UserSchema.loginData.create), loginUser); 
```

@musllim remove all similar comments